lib/aws/assume-role.js

```diff
@@ -46,9 +50,9 @@ function assumeRole (clientName, additionalParameters) {
         }
         if (data.error) {
           if (data.error_description === 'jwt expired') {
-            return reject('Unauthorized, your access token has expired. Please use the login command to login again.');
+            return reject('Unauthorized, your access token has expired. Please login again.');
```

Comment:
I've used the Australian "unauthori**s**ed" in the BMP CLI, but maybe we should just standardise on USA spelling everywhere?